src/platforms/evdev/CMakeLists.txt

```diff
@@ -6,6 +6,12 @@ else ()
   add_compile_definitions(MIR_LIBINPUT_HAS_ACCEL_PROFILE=1)
 endif ()
 
+if ("${LIBINPUT_VERSION}" VERSION_LESS "1.19")
+  add_compile_definitions(MIR_LIBINPUT_HAS_VALUE120=0)
+else ()
+  add_compile_definitions(MIR_LIBINPUT_HAS_VALUE120=1)
+endif ()
+
```

Comment:
I know this is consistent with other `MIR_LIBINPUT...` macros, but IME the usual idiom is:
```
if ("${LIBINPUT_VERSION}" VERSION_GREATER_EQUAL "1.19")
  add_compile_definitions(MIR_LIBINPUT_HAS_VALUE120)
endif ()
```
with
```
#ifdef MIR_LIBINPUT_HAS_VALUE120
...
```
Is there a reason for this approach?

Reply:
No reason 🤷‍♂️ Just following what was done for `MIR_LIBINPUT_HAS_ACCEL_PROFILE`. Now changing this.